### PR TITLE
Cache token

### PR DESCRIPTION
**Pull request by mayqmh**
Caching token request into `/tmp` folder. Tested locally with staging auth.
Should i create a folder for this in nuCloud so the rest can review?

### CODE REVIEW

File: src/Auth0Provider.cs [hunk 1]
```diff
@@ -15,7 +17,7 @@ public class Auth0Provider : ITokenProvider
         private Auth0TokenResponse _cachedToken;
         private DateTime _expiry;
         private bool _isAuthRequired;
-
+        private readonly string _cachedTokenResponse = @"/tmp/tokenResponse.json";
```
Comment (clouddra) [inline src/Auth0Provider.cs:20]: do we need to have a different path for different `Auth0TokenRequest`?

**Review by clouddra — CHANGES_REQUESTED** [review]

File: src/Auth0Provider.cs (hunk 1 above)
Comment (mayqmh) [reply src/Auth0Provider.cs:20]: tmp folder is unique to each lambda function so I don't think there's a need to create different path for different Auth0TokenRequest https://forums.aws.amazon.com/thread.jspa?messageID=610055

Comment (clouddra) [reply src/Auth0Provider.cs:20]: What happens if the service needs to retrieve multiple tokens? In that case we need to do `/tmp/tokens_for_svc_1`, `/tmp/tokens_for_svc_2`. We can solve that by using `/tmp/{Audience}`

**Review by clouddra — CHANGES_REQUESTED** [review]

**Comment by mayqmh** [comment]
shifted to another repo

**Pull request closed without merging**